scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -16,7 +16,7 @@ namespace <%= namespace %>.SDK {
         }
 
         public enum ErrorType {
-            HTTP, GraphQL, UserError, NativePaymentProcessingError
+            HTTP, GraphQL, UserError, NativePaymentProcessingError, WebCheckoutVerificationError
```

Comment:
Is there a reason why we need `WebCheckoutVerificationError`. Could we not use a `NativePaymentProcessingError`?